index/find_component.go

```diff
@@ -284,6 +289,11 @@ func (index *SpecIndex) lookupFileReference(ref string) (*yaml.Node, *yaml.Node,
         query = "$"
     }
 
+    query, err := url.PathUnescape(query)
+    if err != nil {
```

Comment:
Same comment as above about the coverage.

Reply:
allright